plugins/webp-uploads/load.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Plugin Name: WebP Uploads
+ * Plugin URI: https://github.com/WordPress/performance/tree/trunk/plugins/webp-uploads
+ * Description: Creates WebP versions for new JPEG image uploads if supported by the server.
+ * Requires at least: 6.3
+ * Requires PHP: 7.0
+ * Version: 1.0.5
+ * Author: WordPress Performance Team
+ * Author URI: https://make.wordpress.org/performance/
+ * License: GPLv2 or later
+ * License URI: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ * Text Domain: webp-uploads
+ *
+ * @package webp-uploads
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+// Define required constants.
```

Comment:
See other comment:
```suggestion
if ( defined( 'WEBP_UPLOADS_VERSION' ) ) {
	return;
}
```

Reply:
Done in https://github.com/WordPress/performance/pull/1011/commits/a6c1ddb7f219e09faaec4a396f7c9e78de90bdc5